modules/microshift-nw-ipv6-dual-stack-migrating-config.adoc

```diff
@@ -0,0 +1,149 @@
+You can verify that all of the system services and pods to have two IP addresses, one for each family, by using the following steps:
+
+. Retrieve the status of the pods by running the following command:
++
+[source,terminal]
+----
+$ oc get pod -A -o wide
+----
++
+.Example output
++
+[source,text]
+----
+NAMESPACE                  NAME                                      READY   STATUS    RESTARTS        AGE     IP                NODE           NOMINATED NODE   READINESS GATES
+kube-system                csi-snapshot-controller-bb7cb654b-7s5ql   1/1     Running   0               46m     10.42.0.6         microshift-9   <none>           <none>
+kube-system                csi-snapshot-webhook-95f475949-jrqv8      1/1     Running   0               46m     10.42.0.4         microshift-9   <none>           <none>
+openshift-dns              dns-default-zxkqn                         2/2     Running   0               46m     10.42.0.5         microshift-9   <none>           <none>
+openshift-dns              node-resolver-r2h5z                       1/1     Running   0               46m     192.168.113.117   microshift-9   <none>           <none>
+openshift-ingress          router-default-5b75594b4-228z7            1/1     Running   0               2m5s    10.42.0.3         microshift-9   <none>           <none>
+openshift-ovn-kubernetes   ovnkube-master-bltk7                      4/4     Running   2 (2m32s ago)   2m36s   192.168.113.117   microshift-9   <none>           <none>
+openshift-ovn-kubernetes   ovnkube-node-9ghgs                        1/1     Running   2 (2m32s ago)   46m     192.168.113.117   microshift-9   <none>           <none>
+openshift-service-ca       service-ca-5d7bd9db6-qgwgw                1/1     Running   0               46m     10.42.0.7         microshift-9   <none>           <none>
+openshift-storage          lvms-operator-656cd9b59b-8rpf4            1/1     Running   0               46m     10.42.0.8         microshift-9   <none>           <none>
+openshift-storage          vg-manager-wqmh4                          1/1     Running   2 (2m39s ago)   46m     10.42.0.10        microshift-9   <none>           <none>
+----
+
+. Retrieve the networks defined by the OVN-K network plugin by running the following command:
++
+[source,terminal]
+----
+$ oc get pod -n openshift-ovn-kubernetes ovnkube-master-bltk7 -o jsonpath='{.status.podIPs}'
+----
++
+.Example output
+[source,text]
+----
+[{"ip":"192.168.113.117"},{"ip":"2001:db9:ca7:ff::1db8"}]
+----
+
+. Retrieve the networks defined in the node resource by running the following command:
++
+[source,terminal]
+----
+$ oc get pod -n openshift-ingress router-default-5b75594b4-228z7 -o jsonpath='{.status.podIPs}'
+----
++
+.Example output
+[source,text]
+----
+[{"ip":"10.42.0.3"},{"ip":"fd01:0:0:1::3"}]
+----
+
+[NOTE]
+====
+To return to single-stack, you can remove the second entry to the networks and return to the single stack that was configured before migrating to dual-stack.
```

Comment:
```suggestion
To return to single stack, you can remove the second entry to the networks and return to the single stack that was configured before migrating to dual-stack.
```
Suggest removing hyphen since it's being used as a subject here